content/reference-rippled.md

```diff
@@ -2190,17 +2192,18 @@ The request includes the following parameters:
-| `ledger_hash`      | String                                     | _(Optional)_ Use instead of ledger\_index\_min and ledger\_index\_max to look for transactions from a single ledger only. (See [Specifying a Ledger](#specifying-ledgers)) |
-| `ledger_index`     | String or Unsigned Integer                 | _(Optional)_ Use instead of ledger\_index\_min and ledger\_index\_max to look for transactions from a single ledger only. (See [Specifying a Ledger](#specifying-ledgers)) |
-| `binary`           | Boolean                                    | (Optional, defaults to False) If set to True, return transactions as hex strings instead of JSON. |
-| `forward`          | boolean                                    | (Optional, defaults to False) If set to True, return values indexed with the oldest ledger first. Otherwise, the results are indexed with the newest ledger first. (Each page of results may not be internally ordered, but the pages are overall ordered.) |
-| `limit`            | Integer                                    | (Optional, default varies) Limit the number of transactions to retrieve. The server is not required to honor this value. |
+| `ledger_index_min` | Integer                                    | Use to specify the earliest ledger to include transactions from. A value of `-1` instructs the server to use the earliest validated ledger version available. This field is optional if you use `ledger_index_max`, however you can use these fields in tandem to define a range of ledgers. This field is optional and unnecessary if you use `ledger_hash` or `ledger_index`. |
+| `ledger_index_max` | Integer                                    | Use to specify the most recent ledger to include transactions from. A value of `-1` instructs the server to use the most recent validated ledger version available. This field is optional if you use `ledger_index_min`, however you can use these fields in tandem to define a range of ledgers. This field is optional and unnecessary if you use `ledger_hash` or `ledger_index`. |
+| `ledger_hash`      | String                                     | Use to look for transactions from a single ledger only. This field is optional and unnecessary if you use `ledger_index_min`, `ledger_index_max`, or `ledger_index`. (See [Specifying a Ledger](#specifying-ledgers).) |
+| `ledger_index`     | String or Unsigned Integer                 | Use to look for transactions from a single ledger only. This field is optional and unnecessary if you use `ledger_index_min`, `ledger_index_max`, or `ledger_hash`. (See [Specifying a Ledger](#specifying-ledgers).) |
+| `binary`           | Boolean                                    | _(Optional)_ Defaults to `false`. If set to `true`, returns transactions as hex strings instead of JSON. |
```

Comment:
Changed lowercased `False` and `True`. WebSocket and JSON-RPC do not accept init capped Boolean values.